content/en/docs/concepts/cluster-administration/system-traces.md

```diff
@@ -9,7 +9,7 @@ weight: 90
 
 <!-- overview -->
 
-{{< feature-state for_k8s_version="v1.22" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.27" state="beta" >}}
```

Comment:
You will need to update the `content/en/docs/reference/command-line-tools-reference/feature-gates.md` file to indicate that `APIServerTracing` is now beta.